test/java/seedu/addressbook/logic/LogicTest.java

```diff
@@ -79,7 +79,7 @@ private void assertCommandBehavior(String inputCommand,
 
         //Execute the command
         CommandResult r = logic.execute(inputCommand);
-
+        saveFile.save(addressBook);
```

Comment:
You should test the different behaviors for mutating and non-mutating commands, right?

Reply:
what does it mean?